app/api/privy/integrations/discord/interactions.rb

```diff
@@ -0,0 +1,20 @@
+module Privy
+  module Integrations
+    module Discord
+      class Interactions < Grape::API
+        desc 'Receive interactions from Bot'
+
+        post '/interactions' do
+          raw_body = request.body.read
+          interaction_request = ::Discord::Interactions::Request.new(params:, headers:, raw_body:)
+          @resolver = ::Discord::Interactions::Resolvers::Resolver.create_resolver(request: interaction_request)
+          @resolver.call
+
+          template = "integrations/discord/#{@resolver.name}"
+          env['api.tilt.template'] = template
+          status @resolver.response_status
```

Comment:
what happens if we respond a non success status to Discord? Is there any advantage on having the ability to respond with something other than a 200? Can we handle that in any way on Discord?

Reply:
We'll have only two response status scenarios: 200 or 401. There is at least one case in which Discord must know the response status the request delivers, specifically the PING request that Discord sends when connecting to the interactions endpoint